client/src/graphQL.ts

```diff
@@ -0,0 +1,5 @@
+import ApolloClient from 'apollo-boost';
+
+export const client = new ApolloClient({
+    uri: `${window.location.protocol}//${window.location.hostname}:8080/graphql`
+});
```

Comment:
Does this just provide a connection to the Apollo server?

Reply:
Creates an Apollo client which connects to the Spring boot GraphQL server.

Reply:
This is then passed to the ApolloProvider component.